meta-hpe/meta-common/recipes-phosphor/vehci/vehci-gxp-b0-workarround.bb

```diff
@@ -0,0 +1,45 @@
+SUMMARY = "VEHCI GXP B0 Workarround"
```

Comment:
Prefer to conform entire PR to the refactored internal change.  See `host-ehci-owner-reset` as per internal commit 2cdfada6f65a792c93e0ae7b99b615c6f1c773be.  Workarround misspelled

Reply:
I agree. I took inspiration from our 2.8 implementation, and will refactor the code for 2.10 as soon as we got it integrated within 2.9. This would avoid to back-port our work in progress 2.10 to 2.9

Reply:
I agree. I took inspiration from our 2.8 implementation, and will refactor the code for 2.10 as soon as we got it integrated within 2.9. This would avoid to back-port our work in progress 2.10 to 2.9